### PR TITLE
Reading an Inventory record from DB

### PR DESCRIPTION
**Pull request by MarvinTellis**
Added code and testcases for Reading an inventory record from DB.

Unit test results are below
image

### CODE REVIEW

File: service/models.py [hunk 1]
```diff
@@ -31,6 +31,9 @@ class Condition(Enum):
     OPEN_BOX = 2
     USED = 3
 
+    # This is the last value in the enum and is meant to be a default value. Nothing should come after this
```
Comment (IrregularHunterX) [inline service/models.py:34]: Not sure if you're going to get any conflicts because I added this change already?

File: service/routes.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 from flask import Flask, jsonify, request, url_for, make_response, abort
 from service.common import status  # HTTP Status Codes
-from service.models import Inventory
+from service.models import Inventory, Condition
```
Comment (IrregularHunterX) [inline service/routes.py:9]: I meant to make this change in the next sprint

The mainline version of routes.py already has import Condition, maybe you can get rid of that line when you merge?

**Comment by vkrishna1997** [comment]
Looks good to me. You just need to pull the latest master branch and rebase off of it. That should address all of Thomas' comments!

**Review by vkrishna1997 — CHANGES_REQUESTED** [review]
Looks good to me. You just need to pull the latest master branch and rebase off of it. That should address all of Thomas' comments!

**Review by IrregularHunterX — COMMENTED** [review]
Everything else looks good to me, getting the changes from mainline will resolve the comments I made

File: service/routes.py (hunk 2 above)
Comment (MarvinTellis) [reply service/routes.py]: I have fixed that.

**Review by vkrishna1997 — APPROVED** [review]
Looks good to me

**Pull request merged**